### PR TITLE
fix(django 2.0): geo_by_addr should return strings

### PR DESCRIPTION
**Pull request by joshuarli**
Since Django 2.0, some portions of their code including templating no longer coerces bytestrings into strings. Acceptance tests caught most of this, but this is one place that went unnoticed until recently:

image

Further, I removed the ISO-8859-1 stuff as it's leftover from when we used https://github.com/maxmind/geoip-api-python way, way back with Python 2 where they return ISO-8859-1 py2 strings by default. The new library we're using just returns py3 str.

Writing tests for this would be pretty expensive - we'd have to essentially download `GEOIP_PATH_MMDB = "/usr/local/share/GeoIP/GeoIP2-City.mmdb"` every time which maxmind provides freely, but you need some auth tokens and stuff. Can't test this on staging either, we only run `geoipupdate` for production. But it's a pretty trivial fix.

### CODE REVIEW

**Review by wedamija — COMMENTED** [review]
To test this could we include a stripped down version of `GeoIP2-City.mmdb` in our test repo somewhere and just operate off of that?

**Comment by joshuarli** [comment]
> To test this could we include a stripped down version of `GeoIP2-City.mmdb` in our test repo somewhere and just operate off of that?

Hmm, I can look into writing a very small mmdb as a fixture.

**Comment by joshuarli** [comment]
Ok, ended up using https://github.com/VimT/MaxMind-DB-Writer-python to write the entry that's there for 8.8.8.8 (just a random example):

```
>>> from mmdb_writer import MMDBWriter
>>> writer = MMDBWriter()
>>> from netaddr import IPSet
>>> writer.insert_network(IPSet(['8.8.8.0/24']), {'city': {'geoname_id': 5328041, 'names': {'de': 'Beverly Hills', 'en': 'Beverly Hills', 'es': 'Beverly Hills', 'fr': 'Beverly Hills', 'ja': 'ビバリーヒルズ', 'pt-BR': 'Beverly Hills'}}, 'continent': {'code': 'NA', 'geoname_id': 6255149, 'names': {'de': 'Nordamerika', 'en': 'North America', 'es': 'Norteamérica', 'fr': 'Amérique du Nord', 'ja': '北ア メリカ', 'pt-BR': 'América do Norte', 'ru': 'Северная Америка', 'zh-CN': '北美洲'}}, 'country': {'geoname_id': 6252001, 'iso_code': 'US', 'names': {'de': 'USA', 'en': 'United States', 'es': 'Estados Unidos', 'fr': 'États-Unis', 'ja': 'アメリカ合衆国', 'pt-BR': 'Estados Unidos', 'ru': 'США', 'zh-CN': ' 美国'}}, 'location': {'accuracy_radius': 1000, 'latitude': 34.0911, 'longitude': -118.4117, 'metro_code': 803, 'time_zone': 'America/Los_Angeles'}, 'postal': {'code': '90210'}, 'registered_country': {'geoname_id': 6252001, 'iso_code': 'US', 'names': {'de': 'USA', 'en': 'United States', 'es': 'Estados Unidos', 'fr': 'États-Unis', 'ja': 'アメリカ合衆国', 'pt-BR': 'Estados Unidos', 'ru': 'США', 'zh-CN': '美国'}}, 'subdivisions': [{'geoname_id': 5332921, 'iso_code': 'CA', 'names': {'de': 'Kalifornien', 'en': 'California', 'es': 'California', 'fr': 'Californie', 'ja': 'カリフ ォルニア州', 'pt-BR': 'Califórnia', 'ru': 'Калифорния', 'zh-CN': '加州'}}]})
>>> writer.to_db_file('test.mmdb')
>>> import maxminddb
>>> m = maxminddb.open_database('test.mmdb')
>>> m.get('8.8.8.8')
{'city': {'geoname_id': 5328041, 'names': {'de': 'Beverly Hills', 'en': 'Beverly Hills', 'es': 'Beverly Hills', 'fr': 'Beverly Hills', 'ja': 'ビバリーヒルズ', 'pt-BR': 'Beverly Hills'}}, 'continent': {'code': 'NA', 'geoname_id': 6255149, 'names': {'de': 'Nordamerika', 'en': 'North America', 'es': 'Norteamérica', 'fr': 'Amérique du Nord', 'ja': '北ア メリカ', 'pt-BR': 'América do Norte', 'ru': 'Северная Америка', 'zh-CN': '北美洲'}}, 'country': {'geoname_id': 6252001, 'iso_code': 'US', 'names': {'de': 'USA', 'en': 'United States', 'es': 'Estados Unidos', 'fr': 'États-Unis', 'ja': 'アメリカ合衆国', 'pt-BR': 'Estados Unidos', 'ru': 'США', 'zh-CN': '美国'}}, 'location': {'accuracy_radius': 1000, 'latitude': 34.09109878540039, 'longitude': -118.41169738769531, 'metro_code': 803, 'time_zone': 'America/Los_Angeles'}, 'postal': {'code': '90210'}, 'registered_country': {'geoname_id': 6252001, 'iso_code': 'US', 'names': {'de': 'USA', 'en': 'United States', 'es': 'Estados Unidos', 'fr': 'États-Unis', 'ja': 'アメリカ合衆国', 'pt-BR': 'Estados Unidos', 'ru': 'США', 'zh-CN': '美国'}}, 'subdivisions': [{'geoname_id': 5332921, 'iso_code': 'CA', 'names': {'de': 'Kalifornien', 'en': 'California', 'es': 'California', 'fr': 'Californie', 'ja': 'カリフ ォルニア州', 'pt-BR': 'Califórnia', 'ru': 'Калифорния', 'zh-CN': '加州'}}]}
```

**Review by evanpurkhiser — APPROVED** [review]
cool

File: tests/sentry/utils/test_geo.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+from django.test.utils import override_settings
+
+from sentry.testutils.cases import TestCase
+from sentry.testutils.factories import get_fixture_path
+
+
+@override_settings(GEOIP_PATH_MMDB=get_fixture_path("test.mmdb"))
+class TestGeo(TestCase):
```
Comment (wedamija) [inline tests/sentry/utils/test_geo.py:8]: Just want to confirm that this test failed before your changes?

Comment (joshuarli) [reply tests/sentry/utils/test_geo.py:8]: Yep.

**Pull request merged**